views/post.py

```diff
@@ -42,3 +42,27 @@ def get_all_user_posts(url):
         serialized_posts = json.dumps(posts)
 
     return serialized_posts
+
+
+def create_post(post):
+    with sqlite3.connect("./db.sqlite3") as conn:
+        conn.row_factory = sqlite3.Row
+        db_cursor = conn.cursor()
+
+        db_cursor.execute(
+            """ 
+            INSERT INTO Posts (user_id, category_id, title, image_url, content)
```

Comment:
Based on issue ticket #40, I believe all new posts should also include values for publication_date (set to the time that this function executes. i.e., datetime.now() ) and approved (set to 1 by default).

Reply:
Fixed